foodsaving/groups/templates/user_inactive_in_group.mjml

```diff
@@ -0,0 +1,33 @@
+<mjml>
+    <mj-head>
+        <mj-include path="../../base/templates/_base-head.mjml"/>
+        <mj-title>{{ group_name }}</mj-title>
+    </mj-head>
+    <mj-body>
+        <mj-container>
+            <mj-include path="../../base/templates/_header.mjml"/>
+            <mj-section>
+                <mj-column>
+                    <mj-text>
+                        <p>
+			{% trans %}Hey, we're missing you!{% endtrans %}
+		    	</p>
+                        <p>
+                            {% trans %}It looks like you've not visited the <a href="{{ group_url }}">{{ group_name }}</a> group for over {{ num_days_inactive }} days.{% endtrans %}
+			</p>
+                        <p>
+                            {% trans %}If you don't visit the group in the next {{ inactivity_deletion_days }} we will sadly have to remove you from that group. {% endtrans %}
```

Comment:
I'd put this paragraph at the end of the e-mail (below the button). First lure them, then threat them ;)
Maybe also give a reason:

```
If you are not interested in {{ group_name }} anymore, you don't need to click. After {{ inactivity_deletion_days }}, you will get removed from {{ group_name }} to keep the member list up to date.
```

Generally, I'd mention the group_name in the e-mail more often instead of referring to it as "the group".

Reply:
I put it to the following. Having the group name twice in two sentences didn't seem good style.

If you are not interested in "{{ group_name }}" anymore, you don't need to click. After {{ inactivity_deletion_days }}, you will be removed from the group to keep the member list up to date.